jmix-dynattr/dynattr-flowui/src/main/resources/io/jmix/dynattrflowui/view/category/category-list-view.xml

```diff
@@ -75,7 +75,7 @@
                     </columns>
                 </dataGrid>
             </vbox>
-            <vbox id="attributesBox" margin="true" padding="false" height="100%" expand="attributesTable">
+            <vbox id="attributesBox" margin="false" padding="false" height="100%" expand="attributesTable">
```

Comment:
The `margin` attribute for `ThemableLayout` has a `false` default value. It doesn't need to be specified explicitly, it is enough to remove the attribute